std/experimental/allocator/building_blocks/null_allocator.d

```diff
@@ -1,14 +1,5 @@
 module std.experimental.allocator.building_blocks.null_allocator;
 
-/*
-  _   _       _ _          _ _                 _
- | \ | |     | | |   /\   | | |               | |
- |  \| |_   _| | |  /  \  | | | ___   ___ __ _| |_ ___  _ __
- | . ` | | | | | | / /\ \ | | |/ _ \ / __/ _` | __/ _ \| '__|
- | |\  | |_| | | |/ ____ \| | | (_) | (_| (_| | || (_) | |
- |_| \_|\__,_|_|_/_/    \_\_|_|\___/ \___\__,_|\__\___/|_|
-*/
-
```

Comment:
sure, but it lets a couple of people smile if they discover this in the source code.
I don't really mind though ;-)